packages/pigment-css-react/src/utils/isUnitLess.ts

```diff
@@ -1,55 +1,4 @@
-// Copied over from https://github.com/emotion-js/emotion/blob/main/packages/unitless/src/index.js
-// since typings of @emotion/unitless are broken. PR to fix here - https://github.com/emotion-js/emotion/pull/3088
-const unitlessKeys: { [key: string]: 1 } = {
```

Comment:
This was added here since `@emotion/unitless`'s types were broken. Does it work now ?